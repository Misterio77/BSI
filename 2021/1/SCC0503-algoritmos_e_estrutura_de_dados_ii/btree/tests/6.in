search 1
search 3
insert 15,"Gabriel","Piruleta","Sistemas de Informacao",8.40
insert 16,"Joy","Mark","Sistemas de Informacao",6.40
insert 17,"Clara","Randal","Sistemas de Informacao",5.40
insert 18,"Stanley","Tim","Sistemas de Informacao",8.60
insert 19,"Jean","Christopher","Sistemas de Informacao",5.90
insert 20,"Terri","Rachel","Sistemas de Informacao",9.90
search 75
search 8
search 45
search 23
search 15
search 65
search 676
update 18,"Ronaldinho","Gaucho","Monstro Sagrado",8.60
search 121
search 7
search 654
search 18
search 71
search 4798
search 45
exit
