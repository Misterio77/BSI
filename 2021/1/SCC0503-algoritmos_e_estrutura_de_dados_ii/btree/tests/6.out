Registro nao encontrado!
Registro nao encontrado!
Registro nao encontrado!
Registro nao encontrado!
Registro nao encontrado!
Registro nao encontrado!
-------------------------------
nUSP: 15
Nome: Gabriel
Sobrenome: Piruleta
Curso: Sistemas de Informacao
Nota: 8.40
-------------------------------
Registro nao encontrado!
Registro nao encontrado!
Registro nao encontrado!
Registro nao encontrado!
Registro nao encontrado!
-------------------------------
nUSP: 18
Nome: Ronaldinho
Sobrenome: Gaucho
Curso: Monstro Sagrado
Nota: 8.60
-------------------------------
Registro nao encontrado!
Registro nao encontrado!
Registro nao encontrado!
